SoloProjects/clc80/Dashboard/Dashboard/DashboardCard.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DashboardCard.swift
+//  Dashboard
+//
+//  Created by Claudia Maciel on 12/13/20.
+//
+
+import UIKit
+
+class DashboardCard: UIView {
+    //MARK: - Properties
+    var image1 = UIImageView()
+    var image2 = UIImageView()
+    let title = UILabel()
+    let subTitle = UILabel()
+    
+    override init(frame: CGRect) {
+        super .init(frame: frame)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    init(image1: UIImageView, image2: UIImageView, title: String, subTitle: String) {
+        super .init(frame: .zero)
+        self.image1 = image1
+        self.image2 = image2
+        self.title.text = title
+        self .subTitle.text = subTitle
+        configure()
+    }
+    
+    private func configure() {
+        layer.cornerRadius = 10
+        self.backgroundColor = .systemPink
+        
+        self.addSubview(image1)
+        self.addSubview(image2)
+        self.addSubview(title)
+        self.addSubview(subTitle)
```

Comment:
This method has a mix of including `self.` and excluding it. Make sure you pick one style and stick with it, at least within a file or a feature. My recommendation is to exclude `self.` when it can be, that's what most places are doing now.

```suggestion
        layer.cornerRadius = 10
        backgroundColor = .systemPink
        
        addSubview(image1)
        addSubview(image2)
        addSubview(title)
        addSubview(subTitle)
```